interfaces/acados_template/acados_template/generate_solver.py

```diff
@@ -35,6 +35,8 @@
 from .generate_c_code_explicit_ode import *
 from .generate_c_code_implicit_ode import *
 from .generate_c_code_constraint import *
+from .generate_c_code_nls_cost import *
+from .generate_c_code_nls_cost_e import *
```

Comment:
I think this function is not there (yet).
It might be easier to gather the functionality in one function.

Reply:
Thanks! I have pushed the missing file.